rca/enquire_to_study/views.py

```diff
@@ -189,14 +189,32 @@ def send_user_email_notification(self, form):
             fail_silently=False,
         )
 
+    def set_session_data(self, form_data, enquiry_submission):
+        # Sets form data into session variable for analytics.
+        self.request.session["enquiry_form_session_data"] = {
+            "event": "form-submission",
+            "formName": "Enquire to study form",
+            "formId": enquiry_submission.id,
```

Comment:
In the ticket, this is requested as `'formId':'###',` - I'm waiting on a response from Chris danks, but the submission object ID that's created make most sense to me